libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -48,9 +52,8 @@
 <ng-template #empty>
   <div class="empty">
     <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
-        >JavaScript</a
-      >".
+      Try searching for a topic, for example "<a aria-label="Search for JavaScript"
```

Comment:
Is an `a` tag right for this?

Reply:
And again, does this need an aria-label? What about a role?

Reply:
The anchor tag is for _navigating_. Is the behaviour here navigation?

Reply:
I got it - it is not navigating. I updated it to use a `span` tag.